src/main/external-resources/UMS.conf

```diff
@@ -866,6 +866,13 @@ ffmpeg_fontconfig =
 # Default: true
 ffmpeg_mencoder_problematic_subtitles =
 
+# Output silent audio
+# --------------------------------------------
+# Output silent audio when a movie do not have audio in ffmpegvideo. Some of renderers 
+# do not support a movie without audio. This can be perfomed by either UMS.conf or renderer.conf.
+# Default: false
+ffmpeg_output_silent_audio=
```

Comment:
Really minor formatting thing - please add a space with the =, so `ffmpeg_output_silent_audio =`